lib/presentation/pages/presenter.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:healthy_routine_mobile/healthy_routine.dart';
+import 'package:flutter/material.dart';
+
+class Presenter {
+  static String taskName;
+  static Category category;
+  static List<Weekday> listWeekDay = List();
+  static TimeOfDay startTime;
+  static TimeOfDay endTime;
+  static bool repeat;
+
+  static void selectListWeekDay(Weekday day, bool selected) {
+    selected
+        ? listWeekDay.add(day)
+        : listWeekDay.remove(day);
+  }
+
+  static void saveTask() {
```

Comment:
Why static methods and properties?

Reply:
We does not need an instance of this class, i thought this class will only store the data and call the database service.